src/pages/components/tile/usage.mdx

```diff
@@ -366,88 +401,95 @@ the chevron icon.
 <Row>
 <Column colLg={8}>
 
-![Tile anatomy](images/tile-usage-6.png)
+![Tile anatomy](images/tile-usage-8.png)
 
 </Column>
 </Row>
 
-1. **Container**: Includes a title, optional label, and the close icon.
+1. **Image (optional)** : The image follows the aspect ratios with the default
```

Comment:
The image size follows standard Carbon aspect ratios (in this case, with a default size of 4:3)